src/_guides/language/analysis-options.md

```diff
@@ -12,12 +12,37 @@ With the help of the analyzer, you can find
 simple typos. For example, perhaps an accidental semicolon
 made its way into an `if` statement:
 
-{% asset guides/avoid-empty-statements.png alt="`if (count < 10);` results in a hint: Avoid empty statements." %}
+<blockquote class="ml-3" markdown="1">
+<?code-excerpt "analysis/lib/lint.dart (empty_statements)"?>
+```dart
+void increment() {
+  if (count < 10) ; /*1*/
+  count++;
+}
+```
+
+{:.console-output}
+<?code-excerpt "analysis/analyzer-results.txt" retain="empty_statements" replace="/ at (lib|test)\/\w+\.dart:\d+:\d+/ at \/*1*\//g"?>
+```nocode
+lint • Avoid empty statements at /*1*/ • empty_statements
```

Comment:
This part seems weird to me... you'd never see comment text in there. Can we be more realistic?

Reply:
Ok, I'll see if I can think of something. I have an idea; I'll try it out tomorrow.

Reply:
I've added line numbers and dash-underlined in red just like, say, VS Code does. Let me know if that works for you.

Reply:
Updates done and staging server refreshed. PTAL @kwalrath